src/components/project-list/project-table.tsx

```diff
@@ -20,10 +20,17 @@ type Props = {
   footer?: React.ReactNode;
   showDetails?: boolean;
   metricsCell?: (project: BestOfJS.Project) => React.ReactNode;
+  testid?: string;
```

Comment:
Maybe I'm wrong but I find a bit annoying that we change the API of this component to add a property used only for testing.

Do you think we really need it? Usually table of projects are just below a title, could we target the tables using the titles?

E.g. if we need to target the first table of projects in the top page, something like "the table just after the title `Hot projects`?

About the test IDs the [documentation](https://testing-library.com/docs/queries/about#priority) says:

> getByTestId: The user cannot see (or hear) these, so this is only recommended for cases where you can't match by role or text or it doesn't make sense (e.g. the text is dynamic).

Reply:
@michaelrambeau I agree. Let me try to improve it.

Reply:
I removed the `testid` prop. Instead, I use the `hot-projects-section` selector.